api/src/main/java/org/apache/iceberg/rest/CreateNamespaceResponse.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest;
+
+import java.util.Map;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+
+/**
+ * Represents a REST response to a create a namespace / database request.
+ *
+ *
+ */
+public class CreateNamespaceResponse {
+  private Namespace namespace;
+  private Map<String, String> properties;
```

Comment:
Are these the properties that the namespace has after create? For example, this may include a "created-at" property that was added by the server automatically.

Reply:
Yeah that was my thinking (about server side additions). I was thinking that this would be either:
1) Properties the server added, such as `created-at`.
2) The entire properties of the namespace, including the ones the user provided, to give a full picture.